database/cassandra.js

```diff
@@ -0,0 +1,55 @@
+const cassandra = require('cassandra-driver');
+const path = require('path');
+
+const client = new cassandra.Client({
+  contactPoints: ['127.0.0.1:9042'],
+  // host: 'localhost',
+  // port: '9160',
+  localDataCenter: 'datacenter1'
+});
+
+client.connect()
+
+// .then(() => {
+//   const query = `DROP TABLE reviews.hotels1`;
+//     return client.execute(query);
+// })
+
+.then(() => {
```

Comment:
This is really just semantics but you may want to tab the .then in so it's clear it follows the client connection. Not totally important, just a note.